src/Components/TMS/Organisms/FOTMSSidebar/index.tsx

```diff
@@ -0,0 +1,248 @@
+import React, { useEffect, memo } from "react";
+// import Router from "next/router";
```

Comment:
Remove unused code

Reply:
this PR is for header ticket only. please review only for FOHeader.